src/crdaScan.ts

```diff
@@ -83,12 +83,6 @@ export async function crdaScan(
     ghCore.setOutput(Outputs.REPORT_LINK, reportLink);
 
     if (!crdaData.analysed_dependencies) {
```

Comment:
since we have this same logic in the `analyse` function now, is there some way we could check this only in one place?

Reply:
I wanted to it like if analyse function returns `""` or `undefined`(now) return from here

Reply:
I wanted to it like if analyse function returns `""` or `undefined`(now) return from here

Reply:
so the second (outer) `if (!crdaData.analysed_dependencies)` can just be removed i think

Reply:
Yes

Reply:
Yes